labelme/widgets/worker_name_dialog.py

```diff
@@ -30,5 +34,5 @@ def onConfirm(self):
         self.accept()
 
     def write_worker_name(self, worker_name):
-        with open(sys.path[0] + '/worker_name.txt', "a") as f:
+        with open(sys.path[0] + self.split_char +'worker_name.txt', "a") as f:
```

Comment:
os 마다 경로 구분자가 다른 문제는 os.path.join 을 사용하면 해결이 안되나요?
```suggestion
        with open(os.path.join(sys.path[0], 'worker_name.txt'), "a") as f:
```

Reply:
그 방법이 있었군요~! 반영 완료했습니다
감사합니다~!